chain-impl-mockchain/src/testing/ledger.rs

```diff
@@ -104,7 +104,7 @@ impl ConfigBuilder {
             consensus_version: ConsensusVersion::Bft,
             transaction_max_expiry_epochs: None,
             #[cfg(feature = "evm")]
-            evm_params: chain_evm::Config::Istanbul,
+            evm_params: chain_evm::Config::default(),
```

Comment:
Maybe we should also need to somehow configure evm config, and run tests cases on the different versions

Reply:
The tests in [chain-libs/chain-impl-mockchain/src/ledger/evm.rs](https://github.com/input-output-hk/chain-libs/blob/b6590318da760d223022bf4e3b1c42e1ee697242/chain-impl-mockchain/src/ledger/evm.rs#L532) are unit tests. This ones instead are scenario test, "one level up" of those, and meant to test that all the create fragment and apply/run transaction works as well

Yes we have a way to configure the evm config here https://github.com/input-output-hk/chain-libs/blob/7f94a61365c36a30d23108e6ec81780729dff990/chain-impl-mockchain/src/testing/e2e/evm_mapping.rs#L16 
I didnt make different tests for each evm config because in the case of evm_mapping we dont use any of the evm parameters, but it will probably be the case when we test evm_transaction

Reply:
yea, sure. sorry I have confused with the `EvmTransaction`, indeed for the `EvmMapping` we don't actually need to test on different evm configs. Sorry 😅

Reply:
No worries, better safe than sorry 😄